pkgs/by-name/re/reposilite/plugins.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  newScope,
+  reposilite,
+}:
+lib.makeScope newScope (
+  self:
+  let
+    inherit (reposilite) version;
+
+    mkPlugins = lib.mapAttrs (
+      name: hash:
+      self.callPackage (
+        { fetchurl }:
+        fetchurl {
+          url = "https://maven.reposilite.com/releases/com/reposilite/plugin/${name}-plugin/${version}/${name}-plugin-${version}-all.jar";
+          inherit version hash;
+
+          meta = {
+            description = "${name} plugin for Reposilite.";
```

Comment:
This would make it so that the description would start with a lowercase character which is not ideal

Reply:
hopefully resolved by the ugly thing i wrote